include/ht_file.h

```diff
@@ -19,8 +19,6 @@
 #include <ht_file_interface.h> //IFile
 #include <string> //std::string typedef
 #include <cstddef> //size_t typedef
```

Comment:
But ht_file_interface contains the base class (`IFile`) for `File`, so any changes regarding `m_handle`'s type there would have to be made in `File` as well. I guess we should still do it in case `IFile` is removed, but I don't think that will happen.